src/main/java/tan/Deadline.java

```diff
@@ -0,0 +1,42 @@
+package tan;
+public class Deadline extends Task {
+
+    private String dateTime;
+
+    public Deadline(String description, String dateTime) {
+        super(description);
+        this.dateTime = dateTime;
+    }
+
+    /**
+     * Returns the icon for Deadline tasks.
+     *
+     * @return Returns the String "D"
+     */
+    public String getIcon() {
```

Comment:
This method seems rather unnecessary. Perhaps you do not need to have a method to get "D"?